README.md

```diff
@@ -149,18 +149,24 @@ All "locals" are exposed to your view in `this.props`. These should work identic
 
 ```js
 <script dangerouslySetInnerHTML={{__html: `
-  // google analytics
+  // Google Analytics
   // is a common use
 `}} />
 ```
 
 * It's not possible to specify a doctype in JSX. You can override the default HTML5 doctype in the options.
 
+## Contributors
+* [Zpao][zpao] (Author)
+* [Venkat Reddy][svenkatreddy]
+
```

Comment:
Please keep your PRs to a single purpose and do this separately, it's entirely unrelated to switching to babel-env.

Also, please use my name (`Paul O’Shannessy`)

Reply:
noted! removed, will open a separate PR for that